tgui/packages/tgui/interfaces/AdminAntagMenu.js

```diff
@@ -0,0 +1,653 @@
+          ) {
+            return -1 * i;
+          }
+          if (typeof a[sortId] === 'number') {
+            return (a[sortId] - b[sortId]) * i;
+          }
+          return a[sortId].localeCompare(b[sortId]) * i;
+        })
+        .map((objective, index) => (
+          <Table.Row key={index}>
+            <Table.Cell>
+              <Button
+                tooltip={objective.obj_desc}
+                onClick={() =>
+                  act('vv', {
+                    uid: objective.obj_uid,
+                  })
+                }
+              >
+                {objective.obj_name}
+              </Button>
+            </Table.Cell>
+            <Table.Cell>
+              {objective.no_target
+                ? ''
+                : objective.track.length
+                  ? objective.track.map((target, index) => (
+                      <Button
+                        key={index}
+                        onClick={() =>
+                          act('follow', {
+                            datum_uid: target,
+                          })
+                        }
+                      >
+                        {objective.target_name}{' '}
+                        {objective.track.length > 1
+                          ? '(' + (parseInt(index, 10) + 1) + ')'
+                          : ''}
+                      </Button>
+                    ))
+                  : objective.target_name + ' (не найдено)'}
+            </Table.Cell>
+            <Table.Cell>
+              <Box color={objective.status ? 'green' : 'grey'}>
+                {objective.status ? 'Успешно' : 'Незавершено'}
+              </Box>
+            </Table.Cell>
+            <Table.Cell collapsing>
+              <Button
+                onClick={() => {
+                  act('obj_owner', {
+                    owner_uid: objective.owner_uid,
+                  });
+                }}
+              >
+                {objective.owner_name}
+              </Button>
+            </Table.Cell>
+          </Table.Row>
+        ))}
+    </Table>
+  );
+};
+
+const Security = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const { security } = data;
+  const [searchText, setSearchText] = useLocalState(context, 'searchText', '');
+  const [sortId, _setSortId] = useLocalState(context, 'sortId3', 'health');
+  const [sortOrder, _setSortOrder] = useLocalState(context, 'sortOrder', true);
+
+  const getColor = (officer) => {
+    if (officer.status === 2) {
+      return 'red';
+    }
+    if (officer.status === 1) {
+      return 'orange';
+    }
+    if (officer.broken_bone || officer.internal_bleeding) {
+      return 'yellow';
+    }
+    return 'grey';
+  };
+  const getStatus = (officer) => {
+    if (officer.status === 2) {
+      return 'Мертв';
+    }
+    if (officer.status === 1) {
+      return 'Без сознания';
+    }
+    if (officer.broken_bone && officer.internal_bleeding) {
+      return 'Сломана кость, ВК';
```

Comment:
```suggestion
      return 'Сломана кость, внутреннее кровотечение';
```
Если так написать, интерфейс не сломается, всё влезет? Да - меняй на предложенное, нет - забей.

Reply:
Сокращение именно из-за интерфейса и его поломки.

Reply:
Принял, тогда оставь так.